modules/rhcos-enabling-multipath.adoc

```diff
@@ -21,6 +21,12 @@ On IBM Z and LinuxONE, you can enable multipathing only if you configured your c
 
 .Prerequisites
 * You have a running {product-title} cluster that uses version 4.7 or later.
++
+[NOTE]
+====
+{product-title} does not support enabling multipathing as a day-2 activity on nodes that have been upgraded from 4.6 or earlier. 
```

Comment:
I'm not sure if it's right to write "day-2" or "day 2". We seem to do both in the repo now, maybe something we should consider standardizing on at some point. Not to worry about for this PR, but a thought for the future!

Reply:
@bergerhoffer 
I see day-2 in a few places, day 2 doesn't seem to appear, based on atom searches. I can change to _a post-installation task_ if that works better.

Reply:
@mburke5678 I see "Day 2" capitalized in the repo a few places. I honestly don't know what the right answer is here, just thinking it would be good to be on the same page across the repo on whether we use the phrasing day1/day2 or not, and if so, how?